models/comments.model.js

```diff
@@ -17,8 +17,13 @@ exports.selectCommentsByArticleId = (article_id) => {
         });
 };
 
-exports.insertComment = (article_id, username, body) => {
-    // console.log(article_id, username, body, "model");
+exports.insertCommentByArticleId = (article_id, username, body) => {
+    if (!body || body === "") {
+        return Promise.reject({
+            status: 400,
+            msg: "400: Required Key Missing",
```

Comment:
Custom message here is alright, but actually it's a psql error - 23502, can add an if statement in your psql error handler to handle this error